src/main/java/wellnus/reflection/feature/IndexMapper.java

```diff
@@ -0,0 +1,31 @@
+package wellnus.reflection.feature;
+
+import java.util.HashMap;
+import java.util.Set;
+
+/**
+ * Map display index(1...n) onto s set of integers.
```

Comment:
is the 's' in 'onto s set of integers' a typo? did you mean 'a set of integers'?